README.md

```diff
@@ -227,6 +227,16 @@ Example: If used in express, it returns unique store id per request.
 const requestIdentifier = store.getId();
 ```
 
+### getShortId()
+
+Gets the short unique store id created for the current context/scope.
+
```

Comment:
Also add a line saying - This is same as `getId()`; the difference being just that it returns the id of shorter length (8 characters).